src/libs/test.ts

```diff
@@ -0,0 +1,37 @@
+counter = 0;
+
+function incrementCounter() {
+  counter++;
+}
+
+function sum(a: number, b: number) {
+  return a + b;
+}
+
+function multiply(a: number, b: number) {
+  return a * b;
+}
+
+const apiKey = process.env.API_KEY;
+
+function fibonacci(n: number): number {
+  if (n <= 1) {
+    return n;
+  }
+  return fibonacci(n - 1) + fibonacci(n - 2);
+}
```

Comment:
@trag-bot Recursive function without memoization can lead to performance issues due to repeated calculations of the same values.